src/commands/create/contract.ts

```diff
@@ -445,32 +449,38 @@
             const account = userWallet.connect(provider)
 
             // Deploy a metadata renderer contract
-            const renderAbi = JSON.parse(fs.readFileSync(`./src/abi/develop/EditionMetadataRenderer.json`).toString())
-            const rendererFactory = new ContractFactory(renderAbi, EditionMetadataRenderer.bytecode, account)
+            // TODO: this needs to be removed in the future and a reference to the deployed EditionsMetadataRendererProxy needs to be made here
+            const renderAbi = JSON.parse(fs.readFileSync(`./src/abi/develop/EditionsMetadataRenderer.json`).toString())
+            const rendererFactory = new ContractFactory(renderAbi, EditionsMetadataRenderer.bytecode, account)
```

Comment:
So we have the method getABIs that is a large map of contracts. Shold we add that there? This hardcodes develop as well. Which i guess ok for now ,but we should make sure to update this before promoting to testnet.